sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -96,19 +99,26 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
     checkTypePromotion(DateTimeUtils.fromJavaTimestamp(new Timestamp(intNumber.toLong * 1000L)),
         enforceCorrectType(intNumber.toLong, TimestampType))
     val strTime = "2014-09-30 12:34:56"
-    checkTypePromotion(DateTimeUtils.fromJavaTimestamp(Timestamp.valueOf(strTime)),
-        enforceCorrectType(strTime, TimestampType))
+    checkTypePromotion(
+      expected = 1412080496000000L,
```

Comment:
why can't we still use `DateTimeUtils.fromJavaTimestamp(Timestamp.valueOf(strTime)`?

Reply:
look at `QueryTest`, we set timezone to `America/Los_Angeles` for tests. Maybe we should use the same default value here?